Core/Core/Assignments/Assignment.swift

```diff
@@ -207,6 +212,10 @@ extension Assignment {
                 AssignmentDate.save($0, assignmentID: id, in: client)
             })
         }
+
+        if let items = item.overrides {
+            overrides = Set(items.map { AssignmentOverride.save($0, in: client) })
+        }
```

Comment:
You mean delete the previous ones? The relationship is set to cascade on delete. That means the previous ones should get cleaned up automatically, right?

Reply:
I mean if an assignment has overrides once when we sync then the overrides get removed on the backend, when we sync again the overrides will still be set for us locally and not removed.

Reply:
I tried it out and it did not behave the way that I was expecting it would. Seems to be fine. I would expect to have to set `model.overrides` to nil if `item.overrides` is nil but I guess not?